admin/tools/docker/deployment/parallel/run.sh

```diff
@@ -51,6 +51,7 @@ docker run --detach=true \
     -e "QSERV_MASTER=$MASTER" \
     $DATA_VOLUME_OPT \
     $LOG_VOLUME_OPT \
+    -v /etc/localtime:/etc/localtime:ro \
```

Comment:
The procedure has been improved (look at the 'Files Changes' tab of this pull request page) to allow more control in setting both 'timezone' and 'local time'. What's presently implemented is:
- the default timezone of a container is set to Etc/UTC
- when the container starts it will check if the deployment procedure wants it to switch to some other timezone. And if so the script will make proper adjustment. Otherwise it will leave the default timezone as it is.
- the 'run.sh' will obtain and pass the local timezone info down to the container

In principle this may be further improved by adding a configuration option into env.sh. This file is supposed to be tuned to address site-specific requirements (such as a location of a temporary folder, the data (MySQL/MariaDB) directory of Qserv, a location within a host system where Qserv would place its log files, etc.).